dcos/dcos-ui
# DCOS-57484: make Quota header buttons consistent with Services

## Testing
1. Go to the Quota tab in the Services view
2. Click the "+"

A dropdown menu should appear with the items "Run a Service" and "Create a Group"

## Trade-offs



## Dependencies



## Screenshots
Before:
![OverflowMenuInQuotaPgHeader-before](https://user-images.githubusercontent.com/2313998/62741372-a0b17580-ba08-11e9-993e-bf67068c42f8.gif)

After:
![OverflowMenuInQuotaPgHeader-after](https://user-images.githubusercontent.com/2313998/62741379-a5762980-ba08-11e9-87e4-fefd68839834.gif)
